entity-framework/core/querying/related-data.md

```diff
@@ -235,7 +235,7 @@ public class Post
     }
 }
 ```
```

Comment:
Consider: ...which is defined in the [Microsoft.EntityFrameworkCore.Abstractions](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Abstractions/) package. _This package contains a minimal set of types such that there is little impact in depending on it. However, t_o avoid...